lib/ts/recipe/emailpassword/components/themes/styles.css

```diff
@@ -197,3 +191,35 @@
 [data-supertokens~="resetPasswordPasswordForm"] {
     padding-bottom: 20px;
 }
+[data-supertokens~="buttonWithArrow"] {
```

Comment:
Please move these to the root style.css, because it is used in multiple recipes.

Reply:
Done